dataprocess/dataprocess.go

```diff
@@ -31,6 +34,23 @@ func WritePCDToFile(pointcloud pc.PointCloud, filename string) error {
 	return WriteBytesToFile(buf.Bytes(), filename)
 }
 
+// WriteJSONToFile encodes the imu data and then saves it to the passed filename.
+func WriteJSONToFile(linearAcceleration r3.Vector, angularVelocity spatialmath.AngularVelocity, filename string) error {
+	buf := new(bytes.Buffer)
+
+	jsonLinAcc, err := json.Marshal(linearAcceleration)
```

Comment:
one thing to check is if it is better to created a combined json as opposed to writing twice. This can be done by creating an imuData object which is a `map[string]r3.Vector`.

Not necessary just could help with parsing if you start experiencing issues